assets/src/application/ui/layout/espressoTable/EspressoDragSortingTable/index.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
```

Comment:
The logic was extracted from here: https://github.com/ant-design/ant-design/blob/master/components/table/demo/drag-sorting.md

Reply:
And class-based components swapped to functional components.

Reply:
Also `immutability-helper` was replaced with ramda's `update` function.

Reply:
Updated: https://github.com/eventespresso/event-espresso-core/pull/2365/commits/02e8a64125ecfc005d95971319d4b46fa2c88361